website/docs/docs/supported-databases/profile-redshift.md

```diff
@@ -71,6 +73,12 @@ my-redshift-db:
 
 </File>
 
+### Specifying an IAM Profile
+
+:::info New in dbt v0.18.0
+The `iam_profile` config option for Redshift profiles is new in dbt v0.18.0
+:::
+
```

Comment:
```suggestion

When the `iam_profile` configuration is set, dbt will use the specified profile from your `~/.aws/config` file instead of using the profile name `default`
```